src/ogma/features/nodeGrouping.ts

```diff
@@ -155,7 +155,7 @@ export class NodeGroupingTransformation {
    */
   public getVirtualNodesOfTransformation(): NodeList<LkNodeData, LkEdgeData> {
     // @ts-ignore getContext exists on the transformation but hidden by the types
-    return this.transformation.getContext().virtualNodes;
+    return this.transformation.getContext().metaNodes;
```

Comment:
it is a change introduced in ogma 5.1, after checking with Leo we needed to adapt this one

Reply:
do we still need the ts-ignore

Reply:
yes seems like ts is still complaining when I run ts build